doc/api_assets/style.css

```diff
@@ -358,7 +360,6 @@ tt, code {
   color: #040404;
   background-color: #f2f2f2;
   border-radius: 2px;
-  padding: .1em .3em;
```

Comment:
Please don't remove this line, `pre > code` always wins over it in terms of rule precendence and it's still necessary to add padding to inline `<code>` blocks.